src/neo/IO/Json/JArray.cs

```diff
@@ -151,5 +151,14 @@ public override string ToString()
             }
             return sb.ToString();
         }
+
+        public override object ToPrimitive()
+        {
+            ArrayList list = new ArrayList();
+
+            items.ForEach(o => list.Add(o.ToPrimitive()));
+
+            return list.ToArray();
+        }
```

Comment:
If these methods are only used in plugins, it is recommended to put them in plugin extension methods.

Reply:
I can't do this using extension methods because I need to overload the method.